markets/perps-market/test/integration/helpers/commitHelper.ts

```diff
@@ -42,6 +42,7 @@ export const commitOrder: CommitOrderType = (data, chainState) => {
       acceptablePrice: data.acceptablePrice(),
       trackingCode: data.trackingCode,
     });
+    await tx.wait(); // force immediate confirmation to prevent flaky tests due to block timestamp
```

Comment:
this is good to know, saw some flakiness in spot market, could be related to this in case they pop up next time